workflows/dev_deploy.yml

```
@@ -0,0 +1,66 @@
+name: CI/CD Pipeline
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)